backend/src/LibBackend/SqlCompiler.fs

```diff
@@ -206,7 +206,8 @@ let rec lambdaToSql
     lambdaToSql fns symtable paramName dbFields typ e
 
   match expr with
-  | EApply (_, FnName name, args, _) ->
+  | EApply (_, FnName name, _typeArgs, args, _) ->
+    // TODO: should the typeArgs be used for anything here?
```

Comment:
TODO

Reply:
I don't know what you mean by this - do you mean we should only allow the apply if the fn has no typeArgs (i.e. `typeArgs = []`)?

Reply:
Ah, I see your comment from SqlCompiler now - came out of order in github UI